### PR TITLE
RabbitMQ direct routing ui

### PR DESCRIPTION
**Pull request by kbaley**
Connects to #1350 

Adds RabbitMQ direct routing topology to the UI

### CODE REVIEW

File: src/ServiceControlInstaller.Engine/Instances/ServiceControlCoreTransports.cs [hunk 1]
```diff
@@ -59,6 +59,14 @@ public class ServiceControlCoreTransports
                 Matches = name => name.Equals("RabbitMQ", StringComparison.OrdinalIgnoreCase)
                                   || name.Equals("ServiceControl.Transports.RabbitMQ.RabbitMQConventionalRoutingTransportCustomization, ServiceControl.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
                                   || name.Equals("NServiceBus.RabbitMQTransport, NServiceBus.Transports.RabbitMQ", StringComparison.OrdinalIgnoreCase)
+            },
+            new TransportInfo
+            {
+                Name = "RabbitMQ :: Direct Routing Topology",
```
Comment (danielmarbach) [inline src/ServiceControlInstaller.Engine/Instances/ServiceControlCoreTransports.cs]: We used `::` for now as a separator. Not much thought went into it. 

@sergioc what would you say is more common in a selection box?

A dash
A colon
Double-colon that we used

To clarify the scope of this issue is not to change the SCMU UI much. Just to add additional selection to the selection box. Previously we had

> AzureServiceBus
> RabbitMQ
> AzureStorageQueues
> SqlServer
> Msmq

we are now changing the selection to

> AzureServiceBus SEPERATOR Forwarding Topology
> AzureServiceBus SEPERATOR Endpointoriented Topology
> AzureServiceBus SEPERATOR .Net Standard (suggestions are welcome here as well)
> RabbitMQ SEPERATOR Conventional Routing Topology
> RabbitMQ SEPERATOR Direct Routing Topology
> AzureStorageQueues
> SqlServer
> Msmq

SQS

Comment (kbaley) [reply src/ServiceControlInstaller.Engine/Instances/ServiceControlCoreTransports.cs]: Done

**Comment by danielmarbach** [comment]
@szymonpobiega this is a low risk change. I'm going ahead and merge it to avoid unnecessary merge conflicts. We can fix things on develop if required

**Comment by bording** [comment]
Glad to see that "Routing Topology" was used vs. "Topology", based on the way #1350 PoA was worded!

**Pull request merged**